addons/medical_treatment/ACE_Medical_Treatment.hpp

```diff
@@ -570,7 +570,9 @@ class ADDON {
         // The number of doses over maxDose where there is a chance to overdose.
```

Comment:
Two lines above:
`// How many of this type of medication can be in the system before the patient can possibly overdose?`

With the dosage system, that is no longer true, so comment needs to be adjusted, to something similar to:

`// How many doses of this medication can be in the system before the patient can possibly overdose?`